modules/Movecraft/src/main/java/net/countercraft/movecraft/commands/ManOverboardCommand.java

```diff
@@ -57,6 +57,7 @@ public boolean onCommand(CommandSender commandSender, Command command, String s,
         }
 
         player.setVelocity(new Vector(0, 0, 0));
+        player.setFallDistance(0);
```

Comment:
Let's keep changes in separate PRs

Reply:
If by that you mean the manoverboard change, that's a mistake, I'm new to github, I'll remove this file from the PR.